src/librustdoc/clean/mod.rs

```diff
@@ -1473,7 +1473,7 @@ impl<'tcx> Clean<Type> for Ty<'tcx> {
                     let path = external_path(cx, did, false, vec![], empty);
                     inline::record_extern_fqn(cx, did, ItemType::Trait);
                     let bound = PolyTrait {
-                        trait_: ResolvedPath { path, did, is_generic: false },
```

Comment:
This path is for one of the trait bounds after the first (i.e., one of the auto trait bounds) in a `dyn Trait1 + Trait2 + ...` object, so its `Res` should be correct and preserve the `is_generic: false` behavior.